src/include/connection.h

```diff
@@ -166,8 +166,6 @@ struct __wt_layered_table_manager {
  */
 struct __wt_disaggregated_storage {
     char *page_log;
-    char *stable_prefix;
-    char *storage_source;
```

Comment:
Was this unused?

Reply:
After we removed using dir_store in favor of PALI, storage_source is not used.